scripts/release/workflow/publish.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+npx changeset tag
```

Comment:
This is exposing the npm token, we wanted to avoid that. We can keep the command here if we remove the variable from its environment:

```suggestion
# Avoid exposing npm token to dependencies
env -u NPM_TOKEN npx changeset tag
```

Reply:
Though I'm now wondering if it's better to move the command outside. I don't know enough about the security of environment variables.

Reply:
Changed to [this](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/3915/commits/6a2952f6945c8aeb8f3c7cf46551c986b439fec0)